### DISCUSSION TITLE
How to change the size of the pages?

### DISCUSSION THREAD

**Discussion by Ma-Zijing** [academicpages/academicpages.github.io#2014] [Q&A] [answered]
Hi! I want to change the size of the pages (the margins of the pages) so that we can show more words in a line, but I don't find where to set this parameters. Could somebody give some suggestions?

**Comment by rjzupkoii** [comment]
Do you mean making the side bar smaller, or the overall amount of space that the sidebar and main content occupies larger?

**Reply by CaucherWang** [reply]
Both are useful.

**Reply by emilioluissaenzguillen** [reply]
Is it possible to make the side bar wider and the main content narrower? @rjzupkoii

**Reply by rjzupkoii** [reply]
@emilioluissaenzguillen Not easily, there are some settings in `_sass/_themes.scss` that control how wide the main content is and the side bar will adapt to that, but there are a lot of cascading changes that need to be made to get those to work well.

**Reply by emilioluissaenzguillen** [reply]
Thank you @rjzupkoii , and is it possible to swap sidebar and main content, that is, move the side bar to the right and the main content to the left?

**Reply by rjzupkoii** [reply]
@emilioluissaenzguille It's possible, but same as just adjusting the size of things, it's a non-trivial amount of work to actually do it. However, I suspect it's the type of thing that might be toggleable via the `_config.yml` file, so you may want to open an [Issue](https://github.com/academicpages/academicpages.github.io/issues) with the idea since it would make it a lot easier for users that need to localize the template.

**Comment by rjzupkoii** [comment]
I spent a bit of time on this today, and while I'm still learning my way around the code base, it appears that the layout of the sidebar and main content is based upon calculations done when the [SCSS](https://sass-lang.com/) is compiled at deployment time. Using browser developer tools I'm able adjust some of the CSS, but I haven't yet been able to find all of the settings so that it can be done when a website is deployed. There's a chance that this might end up being a non-trivial change to the template to make it an easy setting to adjust.

**Reply by Ma-Zijing** [reply]
Thank you and sry for my late reply! As shown in the figure, I want to use the space in the red box. Specifically, I want the text in the Bio to expand into the red box, making the entire paragraph longer horizontally. Is it possible to realize currently?

![image](https://github.com/academicpages/academicpages.github.io/assets/101317776/b5c478fd-6935-4c85-a5bd-f298c86ff0e3)

**Comment by Zong-Liang** [comment]
> Hi! I want to change the size of the pages (the margins of the pages) so that we can show more words in a line, but I don't find where to set this parameters. Could somebody give some suggestions?

I have the same need, is there a way now?

**Reply by rjzupkoii** [reply, marked as answer]
There's not a way to do it easily since the empty space is variable to allow for different sizes in monitors. Getting it to work easily would likely need some changes to the template itself. (tagging @Ma-Zijing as well)

**Reply by Ma-Zijing** [reply]
Got it. Thank you :)
